solution-base/mongodb/charts/mongodb-sharded/templates/mongos/mongos-dep-sts.yaml

```diff
@@ -84,6 +84,9 @@ spec:
             runAsNonRoot: {{ .Values.securityContext.runAsNonRoot }}
             runAsUser: {{ .Values.securityContext.runAsUser }}
           {{- end }}
+          command:
+            - /bin/bash
+            - /entrypoint/disable-telemetry.sh
```

Comment:
this looks fishy : we are actually _replacing_ the start script with a wrapper which disable telemetry indeed, but also runs the usual start script...

either we find another name ("run-without-telemetry"), or we find an alternate approach (i imagine it does not work with start scripts ; but maybe it could be done in an initContainer? or updating the mongo image?)

Reply:
I vote for the renaming, because changing the image would require more work everytime we need to upgrade mongodb or the charts...

About the usual mongodb, this one is not (yet) upgraded to 4.2, it still uses `mongo`, not `mongosh` hence not affected by this issue. This is indeed something to work on next, but priority is the mongodb we today use in Artesca. Also, we'll need to rework the `mongo-ci` images we use in the various CIs today (still mongodb 3.x!)